rslabel_package/rslabel/ret.py

```diff
@@ -97,12 +101,49 @@ def app(args):
                     break
             src_dir = 'in_progress/validation/'
             dest_dir = 'done/' if complete else 'unvalidated/'
+        elif tar_name in clarification_tars:
+            delete = True
+            complete = True
+
+            dir_name = os.path.dirname(args.annotations)
+
+            # Looping over to see if there are bad images, and see if the tar is complete
+            for annotation in json_contents:
+                try:
+                    ann = annotation['status']
```

Comment:
We can't have the annotation 'status' field be used for clarification _and_ label validation unless we remove the field after clarification. The mark scripts look at the status field for determining where to start marking in the set. Therefore, someone could return a dataset they just checked out without doing anything and the scripts would think it had been fully validated..

Reply:
Introducing new field in JSON increases size of it(which granted is not much), adds extra complexity. It is better re-use already conveniently implemented logic. I have tried returning it and labeling returned, there was no issue with that. Besides the 'status' field is being deleted after clarification.